examples/test/MeasureValueFilter_test.js

```diff
@@ -0,0 +1,67 @@
+// (C) 2007-2019 GoodData Corporation
```

Comment:
What are these commented tests good for? Why is it commented?

Reply:
Oh I can see, TestCafé tests would not work, because it runs against production which does not support measure value filters yes.